src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2494,13 +2492,10 @@ def _get_extension_version_functionapp(functions_version):
     return '~2'
 
 
-def _get_linux_fx_functionapp(is_consumption, functions_version, runtime, runtime_version):
+def _get_linux_fx_functionapp(functions_version, runtime, runtime_version):
     if runtime_version is None:
         runtime_version = RUNTIME_TO_DEFAULT_VERSION_FUNCTIONAPP[functions_version][runtime]
-    if is_consumption:
-        return '{}|{}'.format(runtime.upper(), runtime_version)
-    # App service or Elastic Premium
-    return _format_fx_version(RUNTIME_TO_IMAGE_FUNCTIONAPP[functions_version][runtime][runtime_version])
+    return '{}|{}'.format(runtime.upper(), runtime_version)
```

Comment:
For `dotnet`, it would be nice to set it to `DOTNET` and not `DOTNET|2` or `DOTNET|3`.

Reply:
Ok that makes sense. I've made that change because it's directly related to how we're handling linuxFxVersion. There's also https://github.com/Azure/Azure-Functions/issues/1496 for further dotnet related changes.